engine/apps/telegram/tests/test_keyboard_renderer.py

```diff
@@ -159,19 +159,19 @@ def test_actions_keyboard_silenced(
         [
             InlineKeyboardButton(
                 text="Acknowledge",
-                callback_data=f"{alert_group.pk}:acknowledge:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:0:oncall-uuid{organization.uuid}",
             )
         ],
         [
             InlineKeyboardButton(
                 text="Resolve",
-                callback_data=f"{alert_group.pk}:resolve:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:2:oncall-uuid{organization.uuid}",
             )
         ],
         [
             InlineKeyboardButton(
                 text="Unsilence",
-                callback_data=f"{alert_group.pk}:unsilence:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:5:oncall-uuid{organization.uuid}",
             )
```

Comment:
does it make sense to keep the tests with the action name inside of the `callback_data`, just to ensure that we still support both cases?